src/components/landing/teaser/OppfolgingsdialogTeaser.tsx

```diff
@@ -26,6 +22,18 @@ const StyledSmallText = styled.p`
   letter-spacing: 0.004em;
 `;
 
+const subtitleText = (oppfolgingsplanStatus: StatusPageToDisplay | null) => {
```

Comment:
Et lite navnendringsforslag: kan denne hete `godkjenningsStatus` i stedet for `oppfolgingsplanStatus` for å være konsistent med const navn der denne blir kalt i linje 64?

Reply:
Godt innspill, endrer :)